### PR TITLE
Add `Fukuoka Ruby Award 2020`.

### PR DESCRIPTION
**Pull request by digitalfukuoka**
Please review and merge.

### CODE REVIEW

File: en/news/_posts/2019-10-17-fukuoka-ruby-award-2020.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: news_post
+title: "2020 Fukuoka Ruby Award Competition - Entries to be judged by Matz"
+author: "Fukuoka Ruby"
+translator:
+date: 2019-10-16 00:00:00 +0000
```
Comment (jcserracampos) [inline en/news/_posts/2019-10-17-fukuoka-ruby-award-2020.md]:
```suggestion
date: 2019-10-17 00:00:00 +0000
```

**Comment by digitalfukuoka** [comment]
Thanks for the comment.
 The award was opened on October 16. Is there a problem at this rate?
Best regards

**Comment by jcserracampos** [comment]
@digitalfukuoka the name of the file and the date variable has to be the same.
So you need to change the filename to 2019-10-16.

**Comment by JuanitoFatas** [comment]
@digitalfukuoka I fixed the file name and merged as 074ba02. Thank you for your contribution 👍 
@jcserracampos Thanks for the review 😊 

The post is live at: https://www.ruby-lang.org/en/news/2019/10/16/fukuoka-ruby-award-2020/

**Comment by digitalfukuoka** [comment]
@JuanitoFatas    Thank you so much!!

**Pull request closed without merging**